File: app/src/main/java/jp/kota_ishimoto/youtubeclient/data/exceptions/YoutubeClientRuntimeException.java
package jp.kota_ishimoto.youtubeclient.data.exceptions;

public class YoutubeClientRuntimeException extends RuntimeException {

    public YoutubeClientRuntimeException(String message) {
        super(message);
    }

    public YoutubeClientRuntimeException(String message, Throwable cause) {
        super(message, cause);
    }

    public YoutubeClientRuntimeException(Throwable cause) {
        super(cause);
    }

}

File: app/src/main/java/jp/kota_ishimoto/youtubeclient/data/exceptions/YoutubeClientRuntimeException.kt
package jp.kota_ishimoto.youtubeclient.data.exceptions

open class YoutubeClientRuntimeException : RuntimeException {

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)

}
